backend/core/src/views/update-listing.hbs

```diff
@@ -189,7 +189,7 @@
       .btn-primary {
         background-color: #004445;
         border-color: #004445;
-        color: #ffffff;
+        color: #ffffff !important;
```

Comment:
Do you know what style is overriding this? Instead of using `!important` can we increase the specificity? Maybe `a.btn.btn-primary {}`?

Reply:
its the href styling, but I can't seem to find where the href styling actually happens. increasing the specificity seems to work here.